### DISCUSSION TITLE
How to lower horizontal set of images

### DISCUSSION THREAD

**Discussion by lpinilla** [marp-team/marp#360] [Q&A] [answered]
Hi, I'm trying to put a title with 3 images side-by-side below it.

I can get it working with the following:

```md
---
# Title

![bg horizontal w:300 h:300](https://fakeimg.pl/800x600/0288d1/fff/?text=A)
![bg w:300 h:300](https://fakeimg.pl/800x600/02669d/fff/?text=B)
![bg w:300 h:300](https://fakeimg.pl/800x600/67b8e3/fff/?text=C)
---
```

But the results I get are the following, the images are centered and cover the title.

![image](https://user-images.githubusercontent.com/10984467/197781797-ad9aa044-3d25-4c48-9ac4-234cd809cbeb.png)

So I was trying to lower the images. I've tried playing around with the `top` and `bottom` paramaters inside the `[]` on the images but had no luck. I wanted to know what am I doing wrong or how could it be done.

Thanks!

**Comment by yhatt** [comment, marked as answer]
Simply use inline Markdown images instead of background images (`bg`). Put image syntaxes in a line to align images side-by-side.

```markdown
# Title

![w:300 h:300](https://fakeimg.pl/800x600/0288d1/fff/?text=A) ![w:300 h:300](https://fakeimg.pl/800x600/02669d/fff/?text=B) ![w:300 h:300](https://fakeimg.pl/800x600/67b8e3/fff/?text=C)
```

You also can tweak how to layout images by using [flexbox](https://web.dev/learn/css/flexbox/) through the scoped inline style (https://marpit.marp.app/theme-css?id=scoped-style).

```html
<style scoped>
  p {
    display: flex;
    justify-content: center;
    gap: 30px;
  }
</style>
```

<img width="1118" alt="aligned-images" src="https://user-images.githubusercontent.com/3993388/197791480-3f879fbd-82d7-477e-98d0-742def6245c6.png">

**Comment by lpinilla** [comment]
Nice! thank you, another question: when using the images without the `bg` parameter, I now see a white background instead of the transparent one that the images have, is there something I can do via code to solve this?

**Reply by yhatt** [reply]
👉 #345
